legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/BaseGitLabApi.java

```diff
@@ -66,7 +66,7 @@
     private static final Random RANDOM = new Random();
     private static final Encoder RANDOM_ID_ENCODER = Base64.getUrlEncoder().withoutPadding();
 
-    private static final Pattern PACKAGEABLE_ELEMENT_PATH_PATTERN = Pattern.compile("^\\w++(::\\w++)*+$");
+    private static final Pattern PACKAGEABLE_ELEMENT_PATH_PATTERN = Pattern.compile("^\\w++(::\\w++)*+[\\w$]*+$");
```

Comment:
Let's add some tests for this. In particular, add tests for isValidEntityName, isValidEntityPath, isValidPackagePath, isValidClassifierPath, and isValidPackageableElementPath, which are all related to this. Note that isValidPackagePath should not allow $.

Reply:
Tests are added! To achieve this, isValidPackagePath should not allow $, I modified the validation a little bit. could you please review?  thanks!